pom.xml

```diff
@@ -160,4 +188,11 @@
 			<scope>test</scope>
 		</dependency>
 	</dependencies>
+	<repositories>
```

Comment:
Actually, seems like this can be removed altogether, which is better IMO, as we don't want to rely on uncontrolled registries.